### PR TITLE
chore: add all the operator binary to the operator container

### LINKED ISSUE #1512: Add arm binary inside the operator container
When adding it must create a symlink to the current build arch

### PR DESCRIPTION
**Pull request by sxd**
The idea it's to add all the binary that support both architetures arm64 and amd64 to the operator container for future use in an experimental feature

Closes #1512

### CODE REVIEW

**Review by litaocdl — APPROVED** [review]

**Review by NiccoloFei — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# This builder stage it's only because we need a command
+# to create a symlink and reduce the size of the image
+FROM gcr.io/distroless/static-debian11:debug-nonroot as builder
+ARG TARGETARCH
+
+COPY --chown=nonroot:nonroot --chmod=0755 dist/manager_* .
```
Comment (gbartolini) [inline Dockerfile:6]: Can we place the manager_* files in a 'bin' subfolder or 'multi-arch' folder?

File: Dockerfile [hunk 2]
```diff
@@ -1,3 +1,13 @@
+# This builder stage it's only because we need a command
+# to create a symlink and reduce the size of the image
+FROM gcr.io/distroless/static-debian11:debug-nonroot as builder
+ARG TARGETARCH
+
+SHELL ["/busybox/sh", "-c"]
+RUN mkdir bind
```
Comment (gbartolini) [inline Dockerfile:7]:
```suggestion
RUN mkdir bind
```
```suggestion
RUN mkdir bin
```

File: Dockerfile (hunk 1 above)
Comment (NiccoloFei) [reply Dockerfile]: Fixed.

**Review by gbartolini — APPROVED** [review]

**Pull request merged**